tsconfig-base.json

```diff
@@ -55,16 +55,19 @@
         /* Report errors for fallthrough cases in switch statement. */
         /* Module Resolution Options */
         "moduleResolution": "node" /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */,
-        // "baseUrl": "./" /* Base directory to resolve non-absolute module names. */,
-        // "paths": {},
+        "baseUrl": "./" /* Base directory to resolve non-absolute module names. */,
+        "paths": {
```

Comment:
if these paths are needed for the top level to run tests, please add them there instead of polluting this tsconfig for the individual packages.